### PR TITLE
Databases & Cors

### PR DESCRIPTION
**Pull request by xori**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -25,6 +25,8 @@ repositories {
 
 
 dependencies {
+	compile('org.hibernate:hibernate-agroal:5.3.1.Final')
+	compile('com.h2database:h2:1.4.197')
```
Comment (xori) [inline build.gradle]: If we're not using SQL we don't need these dependencies.

File: src/main/java/com/ace/alfox/ActionController.java [hunk 2]
```diff
@@ -16,10 +16,11 @@
 @RestController
 class ActionController {
 
+    @CrossOrigin(origins = "*")
```
Comment (xori) [inline src/main/java/com/ace/alfox/ActionController.java]: Needs to be removed

File: src/main/java/com/ace/alfox/ActionController.java [hunk 3]
```diff
@@ -16,10 +16,11 @@
 @RestController
 class ActionController {
 
+    @CrossOrigin(origins = "*")
     @RequestMapping(value = "/perform/{command}", method = { RequestMethod.GET, RequestMethod.POST })
     public ResponseEntity home(HttpServletRequest request, @PathVariable String command, @RequestBody Map<String, Object> params) throws IOException {
         HttpSession session = request.getSession(true);
-        String characterId = (String) session.getAttribute("cid");
+        String characterId = "1234";//String) session.getAttribute("cid");
```
Comment (xori) [inline src/main/java/com/ace/alfox/ActionController.java]: Should go back to looking at the session.

File: src/main/java/com/ace/alfox/AlfoxApplication.java [hunk 4]
```diff
@@ -36,12 +42,11 @@ public AlfoxApplication() {
 		}
 	}
 
+	public void addCorsMappings(CorsRegistry registry) {
+	    registry.addMapping("/").allowedOrigins("*");
```
Comment (xori) [inline src/main/java/com/ace/alfox/AlfoxApplication.java]: `registry.addMapping("/**");`

**Review by xori — COMMENTED** [review]
TODO

**Review by coltin — APPROVED** [review]
lgtm

**Comment by coltin** [comment]
Shipped based on TODO comments, let me know if you want me to re-review. Also get Adam to review.

**Pull request merged**